script/job_generator/generate_job_yaml.py

```diff
@@ -287,6 +288,11 @@ def job_variables(job: Dict[str, Tuple[str, str]]) -> Dict[str, str]:
     variables["ALPAKA_CI_CMAKE_VER"] = job[CMAKE][VERSION]
     variables["ALPAKA_BOOST_VERSION"] = job[BOOST][VERSION]
 
+    if job[BUILD_TYPE][VERSION] == CMAKE_DEBUG:
+        variables["alpaka_DEBUG"] = 2
```

Comment:
There is also `alpaka_DEBUG = 1`. Or is this enabled somewhere else?

Reply:
If we want to test both, I need to add the alpaka debug variable to the parameter set of the job generator. Or extend the `BUILD_TYPE` type to `CMAKE_DEBUG1` and `CMAKE_DEBUG2`.

Reply:
I think the parameter set would be the best fit here.